.github/workflows/workflow.yml

```diff
@@ -22,13 +22,14 @@ jobs:
         id: cache
         with:
           path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/main.txt') }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/main.txt', 'requirements/tests.txt') }}
```

Comment:
Pls align with https://github.com/unit8co/u8timeseries/pull/49

Reply:
I split the test requirements so the tests requirement also need to be cached (and installed)